### PR TITLE
Add compiler flag std=-c11

### PR DESCRIPTION
**Pull request by Jafaral**
Signed-off-by: Jafar Al-Gharaibeh <to.jafar@gmail.com>

### CODE REVIEW

**Review by Don-Ward — COMMENTED** [review]
The change to src/h/opengl.h was made by commit
     223c6a6c  11-Feb-2021 | Reduce the number of warnings produced by clang.
I'm wondering why it's showing up as a diff here.

Apart from that, it looks OK.

File: src/h/opengl.h [hunk 1]
```diff
@@ -555,8 +555,7 @@ struct fontsymbol {
 //#define CWIDTH 0.25
 //#define CNEAR 0.25
 //#define CFAR 50000.0
-//#define CHEIGHT(w) \
-//  (CWIDTH*((double)w->window->height)/((double)w->window->width))
+//#define CHEIGHT(w) (CWIDTH*((double)w->window->height)/((double)w->window->width))
```
Comment (Jafaral) [inline src/h/opengl.h:558]: Are you referring to this line @Don-Ward ?

Comment (Don-Ward) [reply src/h/opengl.h:558]: Yes

Comment (Jafaral) [reply src/h/opengl.h:558]: You added `//` at the front of the second line. My compiler was still complaining about "multi line comment".  I move it all to be one line. So this is a new change

Comment (Don-Ward) [reply src/h/opengl.h:558]: OK

**Review by Don-Ward — APPROVED** [review]

**Pull request merged**